packages/internet_detector.vm/tools/chocolateyinstall.ps1

```diff
@@ -11,7 +11,7 @@ New-Item -Path $toolDir -ItemType Directory -Force -ea 0
 VM-Assert-Path $toolDir
 
 # Install pyinstaller 6.11.1 (needed to build the Python executable with a version capable of executing in admin cmd) and tool dependencies ('pywin32')
-$dependencies = "pyinstaller==6.11.1,pywin32"
+$dependencies = "pyinstaller==6.11.1,pywin32,icmplib"
```

Comment:
I think we should use a concrete version of both pywin32 and icmplib to avoid issues with newer versions.